srcpkgs/tagparser/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'tagparser'
+pkgname=tagparser
+version=9.1.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS:BOOL=ON"
+makedepends="c++utilities-devel zlib-devel"
+depends="zlib"
```

Comment:
it is detected:

=> tagparser-9.1.2_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libc++utilities.so.5 <-> c++utilities>=5.2.0_1
   SONAME: libz.so.1 <-> zlib>=1.2.3_1

Reply:
Done